dash/frontend/src/app/modules/private/pages/cluster/kube-bench/kube-bench.component.html

```diff
@@ -40,7 +40,7 @@ <h3 class="error" *ngIf="statusInvalid">{{benchmarkStatus}}</h3>
           </mat-card>
         </div>
 
-        <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 margin-top-10" style="margin: 0; padding-left:0; padding-right:0; height:150px; display: grid;!important; overflow: auto">
+        <div class="col-xs-12 col-sm-12 col-md-6 col-lg-5 margin-top-10" style="margin: 0; padding-left:0; padding-right:0; height:150px; display: grid;!important; ">
```

Comment:
Why are we removing overflow: auto?

Reply:
There was a vertical scroll appeared next to the card when shrinking to a smaller screen size (when card was small enough the title got moved to the next line).

Reply:
Without overflow:auto how does it look now on small screens?

Reply:
I wonder if maybe it should allow horizontal scrolling but not vertical scrolling on small screens